hivedump

```diff
@@ -189,19 +192,21 @@ sub dump_db {
 	#
 	# Scan for partitions on tables
 	#
-	map {
-		if (/CREATE(\s+EXTERNAL)?\s+TABLE\s+([^\(\s]+).*PARTITIONED\s+BY/) {
-			my $table = $2;
-			print STDERR "  . scanning partitions on table $table\n";
-
-			my @partitions = split "\n", qx|$hive cli -e "use $db; show partitions $table;" 2>/dev/null|;
-
-			for my $p (@partitions) {
-				$p =~ s/([^=]+)=(.*)/$1='$2'/;
-				$ddl .= "ALTER TABLE $table ADD PARTITION ($p);\n";
+	unless ($no_partition) {
```

Comment:
Hi, This branch is merged wrongly. You should keep this `unless ($no_partition) {`